app/view/layer/StyleSwitcherRadioGroup.js

```diff
@@ -151,6 +151,32 @@ Ext.define('CpsiMapview.view.layer.StyleSwitcherRadioGroup', {
             } else if (layer.get('isWfs') || layer.get('isVt')) {
 
                 var sldUrl = layer.get('stylesBaseUrl') + newStyle;
+
+                /**
+                 * Helper function to check if at least one style
+                 * has a non-empty label property.
+                 * @param {Array<{label: String}>} styles Styles object from layerConfig to check
+                 */
+                var hasLabels = function (styles) {
```

Comment:
Maybe re-using the `hasLabels` function of `CpsiMapview.view.menuitem.LayerLabels`? Or adding a util function to `CpsiMapview.util.Legend`?